lib/1_is_greater_than_five.rb

```diff
@@ -7,6 +7,6 @@
 def is_greater_than_five(num)
     num = 4
     if num < 5
-        return true
+        return pickle
```

Comment:
`pickle` is not a value that Ruby can parse. `"pickle"` might have worked.

Also know that if you define `num =4` on line 8, this method will *always* return `"pickle"`.

A more error-proof way to test would be -- instead of re/defining `num` at line 8, _call_ your method with the value you're interested in testing (ie execute `is_greater_than_five(4)`). `4` will then be the stand-in value for `num` throughout that expression of the method.